SmartPlaces.Facilities/lib/OntologyMapper/test/OntologyMappingManagerTests.cs

```diff
@@ -348,10 +344,30 @@ public void ValidateTargetOntologyMapping_ReturnsTrue_ForValidOntologyMapping()
             Assert.False(invalidTargets.Any());
         }
 
-        [Fact]
-        public void ValidateTargetOntologyMapping_ReturnsFalse_ForInvalidOntologyMapping()
+        [Theory]
+        [InlineData("DTDLv2.Space.json")]
+        [InlineData("DTDLv3.Space.json")]
+        public void ValidateTargetOntologyMapping_ReturnsTrue_ExhaustiveMapping(string model)
```

Comment:
This method looks identical to ValidateTargetOntologyMapping_ReturnsTrue_ForValidOntologyMapping()?

Reply:
Well aint that odd.. Old work in progress that ended up elsewhere. Good catch.